guide/how-it-works/ecash/introduction.md

```diff
@@ -0,0 +1,290 @@
+
+### Sending and receiving ecash
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/sending-and-receiving@2x.png"
+   alt-text = "An illustration of the process of sending and receiving ecash tokens to various users"
+   caption = "Sending and receiving ecash tokens."
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+When a user sends an ecash token to another user, the wallet verifies the mint’s signature to confirm the token’s authenticity. The mint then checks its database to ensure the token has not been previously spent. If the token is valid, it is destroyed and replaced with a new token, which is then issued to the recipient. This cycle of destroying the old token and issuing a new one prevents double spending while maintaining user privacy and security.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/sending-bitcoin-flow@1x.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/sending-bitcoin-flow@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/sending-bitcoin-flow@2x.jpg"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/sending-bitcoin-flow@2x.jpg"
+   alt-text = "An illustration comparing how on-chain bitcoin, lightning, and ecash handle the sending of bitcoin."
+   caption = ""
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+Sending ecash and receiving can be done via multiple methods to suit different transaction needs. One of the advantages of bitcoin backed ecash is that the tokens are just strings of text, which are very flexible and can be transmitted via any medium or rail. Some unique ways that ecash can be sent and received, which are not possible with on-chain or Lightning transactions, include:
+
+#### Text message, such as email or SMS:
+Users can send ecash by copying and pasting the token string. This is particularly useful for online transactions or when using text based communication platforms.
+
+#### Bluetooth transmission:
+Most applications allow users to share a token via Bluetooth. However, more comprehensive Bluetooth sending and receiving functionalities have not yet been fully developed.
+
+#### QR codes:
+Unlike base chain and Lightning Network QR codes, which only provide directions for where to send Bitcoin, ecash tokens can be embedded within a QR code itself. This means that a user can claim the ecash token by simply scanning the QR code. This method is particularly useful for in person transactions or quick transfers. QR redemption also enables physical bearer assets like paper notes.
```

Comment:
This would actually be very helpful. I've noticed some differences in the Cashu apps that impact scanning interoperability. For example, a QR code generated in Cashu.me can't be scanned by Minibits because Minibits doesn't support animated QRs. I'm assuming the reason has to do with the size of the QR codes. I've seen eNuts be unable to display some tokens in QR form due to their size. Do you know if Fedi has adopted QR Loop for the same reason?

If this seems to be a common need among ecash wallets due to token size, then maybe we should make a note of this in the design best practices section. I don't think we should go as far as recommending the use of animated QRs, but perhaps just include a note explaining that some wallets are using different QR standards like QR Loop to display ecash tokens due to their string size. We could add something like: "If you encounter issues with displaying ecash in QR forms, consider implementing different QR standards such as QR Loop.

Reply:
I’ve been thinking about this, and I think it’s an important implementation detail with a big impact on the UX. Here are a few UX implications that come to mind:

- When a wallet uses a non-animated QR format (like eNuts), some tokens can’t be displayed as a QR code because they’re too large.
- Animated QR codes require the user to hold their phone up to the screen for longer, and introduce variables like display size and speed settings.

I’m wondering if this falls outside the scope of the guide? Typically designers are at the mercy of whichever QR standard the developers implement, and the UX of QR codes doesn’t vary much. What do you guys think? Is this something we should include? If we did,I think it would make sense to include in the general ecash design best practices. WDYT? @GBKS @mouxdesign @rabbitholiness @yashrajd?